packages/components/src/TimeInput.test.tsx

```diff
@@ -21,41 +21,26 @@ function makeSelection(
   return { selectionStart, selectionEnd, selectionDirection };
 }
 
-function typeString(timeInput: ReactWrapper, str: string) {
-  const inputField = timeInput.find('input');
-  for (let i = 0; i < str.length; i += 1) {
-    inputField.simulate('keydown', { key: str.charAt(i) });
-  }
-}
-
-function selectRange(
-  timeInput: ReactWrapper,
-  selectionRange: SelectionSegment
-) {
-  timeInput.find('input').simulate('select', { target: selectionRange });
-}
-
-function selectCursorPosition(timeInput: ReactWrapper, cursorPosition: number) {
-  selectRange(timeInput, makeSelection(cursorPosition, cursorPosition));
-}
-
 it('mounts and unmounts properly', () => {
-  const maskedInput = makeTimeInput();
-  maskedInput.unmount();
+  makeTimeInput();
 });
 
 describe('typing in matches mask', () => {
   function testInput(text: string, expectedResult = text) {
-    const timeInput = makeTimeInput({ value: 0 });
-    selectCursorPosition(timeInput, 0);
-    typeString(timeInput, text);
-
-    const inputField = timeInput.find('input');
-    expect(inputField.prop('value')).toEqual(expectedResult);
-
-    timeInput.unmount();
+    const { unmount } = makeTimeInput();
+    const input: HTMLInputElement = screen.getByRole('textbox');
+    input.setSelectionRange(0, 0);
+    const cursorPosition = 0;
```

Comment:
Why bother putting this in a constant? Why not just use `0` below?
Or if you're going to make it a constant, provide it as a parameter that defaults to zero, and use it in `input.setSelectionRange` above as well.